.github/workflows/lint.yml

```diff
@@ -16,13 +16,14 @@ jobs:
         uses: fkirc/skip-duplicate-actions@v5
         with:
           concurrent_skipping: same_content_newer
-  lint:
-    needs: pre_job
+
+  lint_solidity:
+    needs: check_for_changes
     if: needs.pre_job.outputs.should_skip != 'true'
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: contracts
+        working-directory: ./contracts
```

Comment:
Relative directories gives me a weird smell. I know this probably isn't an issue here, but just want to check for a reason why this changed.